docs/examples/guardrails_with_chat_models.ipynb

```diff
@@ -2,9 +2,64 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Installing hub:\u001b[35m/\u001b[0m\u001b[35m/guardrails/\u001b[0m\u001b[95mlowercase...\u001b[0m\n",
+      "\u001b[2K\u001b[32m[ ===]\u001b[0m Fetching manifestst\n",
+      "\u001b[2K\u001b[32m[=   ]\u001b[0m Downloading dependencies  Running command git clone --filter=blob:none --quiet https://github.com/guardrails-ai/lowercase.git /private/var/folders/c8/jqt82fpx785dpwpp36ljkgm40000gn/T/pip-req-build-iryeqinr\n",
+      "\u001b[2K\u001b[32m[ ===]\u001b[0m Downloading dependencies\u001b[33mWARNING: Target directory /Users/zaydsimjee/workspace/guardrails/docs/examples/.venv/lib/python3.10/site-packages/guardrails/hub/guardrails/lowercase/validator already exists. Specify --upgrade to force replacement.\u001b[0m\u001b[33m\n",
```

Comment:
Separate task, but we should do an existence check on the validator directories and add the --upgrade flag in the cli when necessary to avoid this.